integration-test/src/main/java/org/commonjava/maven/ext/integrationtest/ITestUtils.java

```diff
@@ -196,7 +198,10 @@ static void runLikeInvoker( String workingDir, String url )
     static Integer runCli( List<String> args, Map<String, String> params, String workingDir )
     {
         ArrayList<String> arguments = new ArrayList<>( args );
-        Collections.addAll( arguments, toJavaParams( params ).split( "\\s+" ) );
+        if (params.size() > 0 )
```

Comment:
Very very minor, but I prefer `!isEmpty()` to `size() > 0`. I checked `HashMap` and it literally checks `size == 0`, so they are the essentially the same, but for some things, you could imagine they have different performance. Also, logically I think it makes more sense as to what is actually being tested.

Reply:
Done